packages/form-generator/src/codegen_graphql_form_generation_result.ts

```diff
@@ -33,11 +31,15 @@ export class CodegenGraphqlFormGeneratorResult
         const fd = await fs.open(filePath, 'w+');
         try {
           await fd.writeFile(content);
-          log?.(`File written: ${path.relative(process.cwd(), filePath)}`);
+          filesWritten.push(
```

Comment:
We should just include the file path here. The caller should have full control over what gets printed (ie the `File written:` prefix)